server/blob/blob-api/src/test/java/org/apache/james/blob/api/BlobStoreContract.java

```diff
@@ -175,4 +179,56 @@ default void readShouldReturnBigSavedData() {
 
         assertThat(read).hasSameContentAs(new ByteArrayInputStream(TWELVE_MEGABYTES));
     }
+
+    @Test
+    @Disabled("JAMES-2829 Not supported yet")
+    default void deleteShouldNotThrowWhenBlobDoesNotExist() {
+        assertThatCode(() -> testee().delete(testee().getDefaultBucketName(), blobIdFactory().randomId()).block())
+            .doesNotThrowAnyException();
+    }
+
+    @Test
+    @Disabled("JAMES-2829 Not supported yet")
+    default void deleteShouldDeleteExistingBlobData() {
+        BlobId blobId = testee().save(testee().getDefaultBucketName(), SHORT_BYTEARRAY).block();
```

Comment:
you should put the testee in a local variable for clarity

Reply:
You mean `testee().getDefaultBucketName` ?

Reply:
no, I mean declaring a `testee` local variable instead of calling `testee()` all the time

Reply:
Fixed!